include/CoreGraphics/CGColor.h

```diff
@@ -40,3 +40,5 @@ COREGRAPHICS_EXPORT size_t CGColorGetNumberOfComponents(CGColorRef color);
 COREGRAPHICS_EXPORT const CGFloat* CGColorGetComponents(CGColorRef color);
 COREGRAPHICS_EXPORT CGPatternRef CGColorGetPattern(CGColorRef color);
 COREGRAPHICS_EXPORT CFTypeID CGColorGetTypeID();
+COREGRAPHICS_EXPORT CGColorRef CGColorCreateGenericRGB(CGFloat red, CGFloat green, CGFloat blue, CGFloat alpha);
```

Comment:
Should add the issue number here then, or at least add declaration so it gets tracked as STUB api